packages/@react-spectrum/combobox/src/MobileComboBox.tsx

```diff
@@ -83,6 +86,11 @@ export const MobileComboBox = React.forwardRef(function MobileComboBox<T extends
     }
   };
 
+  let onClose = () => {
+    state.commit();
+    state.close();
```

Comment:
I think it'll already get closed in `state.commit` right?

Reply:
AHHH, yep, that should be handled now with the final else statement in `commit`, good catch